recipes/larray-editor/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "larray-editor" %}
+{% set version = "0.26" %}
```

Comment:
Shouldn't that be updated to 0.26.1 (or even 0.27)?

Reply:
It would be nice to test the update to version 0.27 with the new `releaser` (when it's ready).

Reply:
But OK to update current meta.yaml with version 0.26.1